src/SmartComponents/PatchSetWizard/InputFields/DescriptionField.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { FormGroup, TextInput } from '@patternfly/react-core';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
+
+const DescriptionField = (props) => {
+    const { input } = useFieldApi(props);
+    const formOptions = useFormApi();
+    const values = formOptions.getState()?.values;
```

Comment:
Is it not possible to use `props.value(s)`, like in the second example here: https://data-driven-forms.org/hooks/use-field-api#usageofinput

Reply:
I am not sure if the props would always have the value when user comes back to the input from next step.

console.log(props) does not list the value in that ^ senario.

![image](https://user-images.githubusercontent.com/59481011/154976994-26c3ba54-1939-4ae8-bfd1-6fe2e5653102.png)